datacenter/dtr/2.2/guides/admin/configure/job-runner.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Jobrunner
+description: Learn about the inner-workings of the jobrunner container in the DTR workflow.
+keywords: docker, job, runner
+---
+
+The jobrunner is presented as a container in DTR. It is a DTR mechanism that:
+
+1. Consumes jobs from a universal jobs queue 
+2. Performs the work of the given action
+
+Having multiple replicas in your cluster increases the number of jobrunner 
+containers, which will increase the number of concurrent jobs.
+
+## Jobrunner Workflow
+The workflow of the jobrunner can be depicted as follows:
+
+[//]: # (Placeholder for jobrunner diagram. @sarahpark will work on the diagram)
+
+When a particular action is performed (see [Job Actions](#job-actions) below) the 
+respective service generates a job that is put onto the jobs queue with an initial 
+status of `waiting`. When a jobrunner worker is available to pick up the action, 
+it will claim the job (i.e: the workerID will be set to the replica ID of the jobrunner
+container that claimed the job) and set the job status to `running`. The worker will 
+carry out the action and then set the appropriate status when complete
+(see [Job Statuses](#job-statuses) below).
+
+### Job Actions
+The available job actions are:
+
+- `gc`
+- `sleep`
+- `false`
+- `tagmigration`
+- `bloblinkmigration`
+- `license_update`
+- `nautilus_scan_check`
+: An image security scanning job. This job does not perform the actual scanning, rather it
+spawns `nautilus_scan_check_single` jobs (one for each layer in the image). Once
+all of the `nautilus_scan_check_single` jobs are complete, this job will terminate.
+- `nautilus_scan_check_single`
+: A security scanning job for a particular layer given by the `parameter: SHA256SUM`. This job
+breaks up the layer into components and checks each component for vulnerabilities
+(see [Security Scanning](../../user/manage-images/scan-images-for-vulnerabilities.md)).
+- `nautilus_update_db`
+: A job that is created to update DTR's vulnerability database. It uses an
+Internet connection to check for database updates through `https://dss-cve-updates.docker.com/` and
+updates the dtr-scanningstore container if there is a new update available (see [Set up vulnerability scans](set-up-vulnerability-scans.md)). 
+- `webhook`
+
+#### Job Capacities
+As mentioned above, each jobrunner container acts as one worker that can carry out these actions.
+The number of a particular action a worker can carry out is defined by it's capacity which can be 
```

Comment:
Since we have this capacity preset, do we want to show users how it can be changed as well?